j2s7s300:
  test_controller_ros:
    joints:
    - j2s7s300_joint_1
    - j2s7s300_joint_2
    - j2s7s300_joint_3
    - j2s7s300_joint_4
    - j2s7s300_joint_5
    - j2s7s300_joint_6
    - j2s7s300_joint_7
    type: ucb_jaco_control/TestControllerROS
    